.github/workflows/build.yml

```diff
@@ -40,3 +44,12 @@ jobs:
 
     - name: Test-CC-Samples
       run: BUILD_MODE=${{ matrix.build_type }} make run-cc-tests
+
+    - name: Build-and-Run-C-Samples-with-LOC
+      run: BUILD_MODE=${{ matrix.build_type }} make clean && BUILD_MODE=${{ matrix.build_type }} CC=gcc LD=g++ L3_LOC_ENABLED=1 make run-c-tests
```

Comment:
Retest the sample programs with `L3_LOC_ENABLED=1` to exercise and test this L3-LOC integration.

Here, we just directly try out `run-c-tests` target, which should subsume the build targets, too.

In a way we are exercising and testing another part of the `Makefile` capability.